corehq/apps/api/resources/v0_5.py

```diff
@@ -489,3 +493,71 @@ def dehydrate(self, bundle):
         bundle.data['product_name'] = bundle.obj.sql_product.name
         bundle.data['transaction_date'] = bundle.obj.report.date
         return bundle
+
+
+class UserDomain(object):
+
+    def __init__(self, domain_name='', project_name=''):
+        self.domain_name = domain_name
+        self.project_name = project_name
+
+
+class UserDomainsResource(Resource):
```

Comment:
@czue I'm not sure about this part. Is there a better way to do this in tastypie?